source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -81,6 +81,11 @@ def getSummaryStatsFromCAdvisor(winNode)
       return getResponse(winNode, relativeUri)
     end
 
+    def getCongifzCAdvisor(winNode)
+      relativeUri = "/configz"
```

Comment:
Does this endpoint work in windows?

Reply:
yes. I've tested that scenario. Windows daemonset also sends the proper values.

Reply:
sent a DM with my cluster running the images from this branch

Reply:
Cool.. thanks!

Reply:
I'm resolving this comment since I've answered it and we need to resolve all comments before merging